website/docs/reference/warehouse-profiles/snowflake-profile.md

```diff
@@ -126,7 +126,7 @@ The "base" configs for Snowflake targets are shown below. Note that you should a
 | query_tag | No | A value with which to tag all queries, for later searching in [QUERY_HISTORY view](https://docs.snowflake.com/en/sql-reference/account-usage/query_history.html) |
 
 ### account
-For AWS accounts in the default US West region, this will be something like `abc123` (without any other segments). For AWS accounts *not* in the default US West region and for GCP and Azure-based accounts, you also have to append the cloud platform, such as `aws`, `gcp` or `azure`, respectively (for example: `abc123.us-central1.gcp`) See [Snowflake's documentation](https://docs.snowflake.com/en/user-guide/intro-regions.html#specifying-region-information-in-your-account-hostname) for more information.
+For AWS accounts in the default US West region, this will be something like `abc123` (without any other segments). For AWS accounts *not* in the default US West region you have to append the region (for example: `abc123.eu-west-1`). For GCP and Azure-based accounts, you also have to append the cloud platform, such as `gcp` or `azure`, respectively (for example: `abc123.us-central1.gcp`) See [Snowflake's documentation](https://docs.snowflake.com/en/user-guide/intro-regions.html#specifying-region-information-in-your-account-hostname) for more information.
```

Comment:
@amychen1776 I am seeing something different in the [Snowflake docs](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html#label-account-locator), which makes me think the AWS example should be as my edits suggest. Can you confirm?
```suggestion
For AWS accounts in the default US West region, this will be something like `abc123` (without any other segments). For AWS accounts *not* in the default US West region you have to append the region and cloud platform. For example, `abc123.eu-west-1.aws`. For GCP and Azure-based accounts, you have to append the region and cloud platform, such as `gcp` or `azure`, respectively. For example, `abc123.us-central1.gcp`. See [Snowflake's documentation](https://docs.snowflake.com/en/user-guide/intro-regions.html#specifying-region-information-in-your-account-hostname) for more information.
```

Reply:
@runleonarun Looking at the [docs](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html#account-identifier-formats-by-cloud-platform-and-region), (and taking into more region consideration), I think it's dependent on the AWS Region. There are still a few that don't need the `.aws` appended to the end of their region id. The list includes US West, US East (N Virginia), EU(Ireland), and two in Asia Pacific.